sklearn/metrics/cluster/supervised.py

```diff
@@ -710,7 +710,8 @@ def normalized_mutual_info_score(labels_true, labels_pred):
     Normalized Mutual Information (NMI) is an normalization of the Mutual
     Information (MI) score to scale the results between 0 (no mutual
     information) and 1 (perfect correlation). In this function, mutual
-    information is normalized by ``sqrt(H(labels_true) * H(labels_pred))``
+    information is normalized by ``sqrt(H(labels_true) * H(labels_pred))``.
+    Note that there exist alternate definitions.
```

Comment:
Not sure that this is so much necessary there.

Reply:
Well, the very popular "IR Book" for example uses a different definition:
https://nlp.stanford.edu/IR-book/html/htmledition/evaluation-of-clustering-1.html
So I am actually inclined that eventually, there should be a parameter to allow using both versions. But at the minimum, it should be pointed out that there is more than one definition of NMI, so results computed with sklearn may not match NMI results computed with other tools.

Reply:
Maybe we should link to wikipedia. There are many more definition than I was aware of. But this should probably go into the user guide instead of here?